src/styles/emets/placeholder.scss

```diff
@@ -0,0 +1,13 @@
+%textContactNam {
+  color: $colorContact;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 140%;
+}
+%textLink {
+  font-weight: 400;
+  padding-top: 8px;
```

Comment:
you have this paffing for elements of contact block,this elements should have margins bottom